src/main/java/org/jboss/processing/StaleIssueCollector.java

```diff
@@ -30,9 +32,9 @@ private static final class State {
         List<SingleIssueState> issueStates = new CopyOnWriteArrayList<>();
     }
 
-    public static StaleIssueCollector getInstance(JiraEndpoint jiraEndpoint) {
-        // TODO add stale period for SearchBuilder.endDate
-        SearchQuery searchQuery = SearchQuery.builder().projects("WFLY").assigneeNotEmpty()
+    public static StaleIssueCollector getInstance(JiraEndpoint jiraEndpoint, LotteryConfig lotteryConfig) {
+        SearchQuery searchQuery = SearchQuery.builder().projects("WFLY")
```

Comment:
this should be eventually configurable

Reply:
That's for sure, it will be loaded from config file when needed. This is just a working demo until we implement some more appropriate logic, so we can see what it does